ShipEngineSDK/Api/AccountApi.cs

```diff
@@ -192,9 +192,10 @@ public async Task<GetAccountSettingsImagesResponseBody> CreateAccountImage(HttpC
 
         requestOptions.Operation = "AccountApi.CreateAccountImage";
 
-        var result = await SendHttpRequestAsync<GetAccountSettingsImagesResponseBody>(HttpMethods.Post, requestOptions, methodClient, _config, cancellationToken);
-
-        return result;
+        var (data, response) = await GetHttpResponse<GetAccountSettingsImagesResponseBody>(HttpMethods.Post, requestOptions.FullPath(), requestOptions.Data, methodClient, _config, cancellationToken);
+        var headers = response.Headers.ToDictionary(h => h.Key, h => h.Value.FirstOrDefault(),
+            StringComparer.InvariantCultureIgnoreCase);
+        return new ShipEngineResponse<GetAccountSettingsImagesResponseBody>(data, response.StatusCode, headers);
```

Comment:
Would all this be worth doing inside the client?  I know this is autogenerated code so the repetition isn't a big deal, but it seems like core functionality that all consumers do.

Reply:
Yes, it would be better done in the client. Now that I am using a new method `GetHttpResponse` anyway, that should be possible. (originally tried to do it using the existing SendHttp...)